apps/dashboard-service/src/dashboard/worker/worker.processor.ts

```diff
@@ -0,0 +1,30 @@
+import { Processor, WorkerHost } from "@nestjs/bullmq";
+import { TotalSectionHeaderService } from "../dto/total-section-header.service";
+import { Job } from "bullmq";
+import { TMLogger } from "@terramatch-microservices/common/util/tm-logger";
+import { DashboardQueryDto } from "../dto/dashboard-query.dto";
+import { CacheService } from "../dto/cache.service";
+import { DelayedJob } from "@terramatch-microservices/database/entities";
+@Processor("dashboard")
+export class WorkerProcessor extends WorkerHost {
```

Comment:
This could use a more specific name. `DashboardProcessor`, or better yet, making this system more generic and putting in in the `common` lib with something like `DelayedJobProcesor`. This won't be the only service that needs to process delayed jobs in the end.